x/data/simulation/operations.go

```diff
@@ -0,0 +1,378 @@
+package simulation
+
+import (
+	"crypto"
+	"fmt"
+	"math/rand"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/codec"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	"github.com/cosmos/cosmos-sdk/simapp/helpers"
+	simappparams "github.com/cosmos/cosmos-sdk/simapp/params"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+
+	"github.com/regen-network/regen-ledger/x/data"
+)
+
+// Simulation operation weights constants
+const (
+	OpWeightMsgAnchor           = "op_weight_msg_anchor"
+	OpWeightMsgAttest           = "op_weight_msg_attest"
+	OpWeightMsgDefineResolver   = "op_weight_msg_define_resolver"
+	OpWeightMsgRegisterResolver = "op_weight_msg_register_resolver"
+
+	ModuleName = "data"
+)
+
+var (
+	TypeMsgAnchor           = data.MsgAnchor{}.Type()
+	TypeMsgAttest           = data.MsgAttest{}.Type()
+	TypeMsgDefineResolver   = data.MsgDefineResolver{}.Type()
+	TypeMsgRegisterResolver = data.MsgRegisterResolver{}.Type()
+)
+
+const (
+	WeightAnchor           = 100
+	WeightAttest           = 100
+	WeightRegisterResolver = 100
+	WeightDefineResolver   = 100
+)
+
+// WeightedOperations returns all the operations from the data module with their respective weights
+func WeightedOperations(
+	appParams simtypes.AppParams, cdc codec.JSONCodec,
+	ak data.AccountKeeper, bk data.BankKeeper,
+	qryClient data.QueryClient) simulation.WeightedOperations {
+
+	var (
+		weightMsgAnchor           int
+		weightMsgAttest           int
+		weightMsgDefineResolver   int
+		weightMsgRegisterResolver int
+	)
+
+	appParams.GetOrGenerate(cdc, OpWeightMsgAnchor, &weightMsgAnchor, nil,
+		func(_ *rand.Rand) {
+			weightMsgAnchor = WeightAnchor
+		},
```

Comment:
it seams that this function is not needed (we could use a noop function). Why don't we set the params directly in lines 52-55?